lib/generator.js

```diff
@@ -1,38 +1,159 @@
 'use strict';
 
+const common = require('metarhia-common');
+
+const structures = [
```

Comment:
These should be `primitives` I think (https://developer.mozilla.org/en-US/docs/Glossary/Primitive).